enable/abstract_window.py

```diff
@@ -552,13 +554,6 @@ def _paint(self, event=None):
 
         self._update_region = []
 
-    def __getstate__(self):
-        attribs = ("component", "bgcolor", "overlay", "_scroll_origin")
```

Comment:
`_scroll_origin` is set to (0,0) in the `__init__` method, but I don't see it used anywhere.  A search of the chaco and enable codebases yields 0 results.  Perhaps it can just be removed entirely?

Reply:
Yeah, it should probably just be removed. I also don't see it being used in some older internal codebases.